com.onesignal.unity.android/Editor/OneSignalConfig.androidlib/consumer-proguard.pro

```diff
@@ -0,0 +1,3 @@
+-keep class com.onesignal.** { *; }
+
+-keep class kotlinx.coroutines.android.AndroidDispatcherFactory {*;}
```

Comment:
Can we add a comment to this line that indicate we are working around an issue with this library?